ops/op2/README.md

```diff
@@ -71,6 +71,20 @@ used if you really know what you are doing.
 Lazy `async(deferred)` calls _may_ be fastcalls, though the resolution will
 still happen on a slow path.
 
+## fastcalls
+
+`op2` requires fastcall-compatible ops to be annotated with `fast`. If you wish
+to avoid fastcalls for some reason (this is unlikely), you can specify `nofast`
+instead.
+
+You may also choose an alternate op function to use as the fastcall equivalent
+to a slow function. In this case, you can specify `fast(op_XYZ)`. The other op
```

Comment:
Could we go with `fast(slow_fallback = op_XYZ)` instead?

Reply:
Slow ops can have multiple fast versions (rather than the other way around), so this way makes more sense, especially if we get multiple overrides.

Reply:
Also, I struggled a bit with getting `x = y` parameters to work with the current setup. I think the parser needs a bit of work if we want to support that style of parameter.

Given that there's just the one op right now, I don't know if it's worth working on that until we have a more useful use case for op args of that style.